Flutter.md

```diff
@@ -0,0 +1,15 @@
+# Native Application development using Flutter Related Resources
+
+## For beginner
+1. [Flutter Documentation](https://flutter.io/)
+2. [Build Native Mobile Apps with Flutter : Udacity](https://in.udacity.com/course/build-native-mobile-apps-with-flutter--ud905)
+3. [Flutter Weekly: A Newsletter](https://flutterweekly.net/)
+3. [mTechViral Flutter Tutorials](https://youtu.be/qWL1lGchpRA)
+
+## For People with Intermediate Level Expertise
+1. [Flutter with Firebase](https://codelabs.developers.google.com/codelabs/flutter-firebase/index.html/)
+
+
```

Comment:
remove extra lines and keep the template as is (put the expert level heading in place)